en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/transport-security.md

```diff
@@ -1,26 +1,133 @@
+|verifyHostName|Verify the backend hostname with the certificate SAN (Subject Alternative Name)|`true`|
+|disableSslVerification|Disable SSL verification with backend clusters|`false`|
+
+### Add a Certificate to Choreo Connect Router as a Trusted Certificate
+Choreo Connect supports certificates in `pem` format. The trusted certificates are added as a single file which contains all the certificates.
+To add a new certificate please follow the steps below.
+
+!!!Important
+    Mount the default certificate location of the Choreo Connect router in `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect or choreo-connect-with-apim/docker-compose.yaml` file. This will allow modifying the certificates easily without loggin to the container.
+
+    ``` tab="Example"
+    router:
+        image: wso2/choreo-connect-router:0.9.1-SNAPSHOT
+        logging:
+        options:
+            max-size: "20m"
+            max-file: "5"
+        environment:
+        ...
+        - ENFORCER_CA_CERT_PATH=/home/wso2/security/truststore/mg.pem
+        volumes:
+        - ../resources/router/security:/home/wso2/security
+        - <PATH>/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt
+    ```
+
+    If you need to change the location of the volume mount, the `trustedCertPath` value in config.toml should also be changed.
+
+1. Convert the certificate into pem format
+ 
+    ```tab="Format"
+        openssl x509 -in <INPUT CERTIFICATE> -out <OUT_PUT_FILE_NAME>.pem
+    ```
+
+    ```tab="Example"
+    openssl x509 -in backend.cert -out backend.pem
+    ```
+
+2. Open the converted backend.pem file and copy the certificate content. (along with `-----BEGIN CERTIFICATE-----` and `-----END CERTIFICATE-----`)
+
+3. Open the default certificate (the mounted certificate) and append the new certificate content and save the file.
+4. Restart the Router container.
+    `docker restart <router_container_name>`
+
+
+## Adding Certificates to Specific Cluters
+
+The above section explains how a certificate can be added to the Choreo Connect router's global trusted certificates. But, if the certificate should be adde only to a specific API backend, it could be done as follows.
+
+#### via API Manager
+
+   Please follow [Adding a Certificate for an Endpoint]({{base_path}}/design/endpoints/certificates/#adding-a-certificate-for-an-endpoint)
+
+<!-- #### via API CTL project
+
+    Please follow -->
+## Securing Communnication between the Components
+
+Communication between internal components of Choreo Connect (adapter, enforcer and router) are secured via mutual ssl.
+
+Each component has it's private key and truststore and it's configured in against each component tag `config.toml` file as below.
+
+```toml
+    [adapter.keystore] 
+    certPath = "/home/wso2/security/keystore/mg.pem"
+    keyPath = "/home/wso2/security/keystore/mg.key"
+
+    [adapter.truststore]
+    location = "/home/wso2/security/truststore"
```

Comment:
Let's mention that truststore is directory. And multiple certificates could be mounted to that directory.